doc/source/security/vault.rst

```diff
@@ -9,7 +9,9 @@ Vault <https://www.hashicorp.com/products/vault/data-protection>`__.
 
 .. important::
 
-  The ``keyring_vault`` plugin works with kv secrets engine version 1.
+  The ``keyring_vault`` plugin works with KV Secrets Engine - version 1 <https://www.vaultproject.io/docs/secrets/kv/kv-v1.html>`__ and
```

Comment:
`version 1` and `Version 2` register inconsistency.

Reply:
Fixed.